test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -155,4 +161,28 @@ describe('POST /chat', () => {
     expect(message.message.id).to.exist;
     expect(memberWithNotification.newMessages[`${group._id}`]).to.exist;
   });
+
+  it('Returns an error when the user has been posting too many messages', async () => {
```

Comment:
Unless a mocked timer is used, I believe these tests may fail if they are delayed (for whatever reason) in the middle of a test.  I'm not sure if the conditions under which each test is run are guaranteed, or if a context switch could occur in the middle of a test run, potentially making the timings significantly longer than expected.